src/lib/aws.ts

```diff
@@ -135,14 +132,18 @@ export async function runECSFargateTask(
     const command = new RunTaskCommand(runTaskInput)
     console.log('AWS: START: Prompting an ECS task to run ...')
     const result = await client.send(command)
-    console.log('AWS:   END: RunTaskCommand finished')
+    /* v8 ignore next 3 */
+    console.log(
+        `AWS:   END: RunTaskCommand finished for tasks`,
+        result.tasks?.map((task) => task),
```

Comment:
What does the output here end up looking like since `task` is an object? Just want to confirm we don't need to serialize something to string or maybe print the ARN :man_shrugging: .

Reply:
thanks for waiting  - it took me a while to find out what actually gets printed. looks like `taskArn` is better data on this than `task` (or we would need to serialize it.) i've made that change